build/helm/keylime/templates/configmap.yaml

```diff
@@ -6,11 +6,15 @@ metadata:
   labels:
     {{- include "keylime.labels" . | nindent 4 }}
 data:
-
-{{- if .Values.global.database.mysql.enable }}
+{{- if or .Values.global.database.mysql.enable .Values.global.database.mysql.external }}
```

Comment:
Is external part needed here? I guess enable is required, then external or internal is used.

My point is I don't see the configuration applies if enable is false, and in the case of enable=false and external=true, it is being configured

Reply:
Yes, this is intended. enable in the existing code means that the mysql hem chart is used.